lib/controllers/github.js

```diff
@@ -68,5 +68,10 @@ module.exports = Router()
     //require req.user
     //get data about user and send it as a json
     res.json(req.user);
+  })
+
+  .delete('/sessions', (req, res) => {
```

Comment:
the route to log a user out does not have to be
``` /api/v1/github/sessions ```
for this assignment, the log out route should be a DELETE route to ``` /api/v1/github ```